tox.ini

```diff
@@ -25,9 +23,9 @@ deps =
     pytest
     pytest-cov
 commands =
-    pytest -m "not rapidjson"
+    pytest -m "not rapidjson" {posargs}
```

Comment:
Wait.. why does `not rapidjson` get `{posargs}` and `rapidjson` does not?

Reply:
and `-s` is already a flag used by Tox

Reply:
Yes it does. I use it quite often when debugging problems... When I do `tox -- -s` in this branch I do get

```
tox -- -s
GLOB sdist-make: /home/mlenders/Repositories/RIOT-OS/riotctrl/setup.py
py35-test create: /home/mlenders/Repositories/RIOT-OS/riotctrl/.tox/py35-test
SKIPPED: InterpreterNotFound: python3.5
py35-rapidjson create: /home/mlenders/Repositories/RIOT-OS/riotctrl/.tox/py35-rapidjson
SKIPPED: InterpreterNotFound: python3.5
py36-test create: /home/mlenders/Repositories/RIOT-OS/riotctrl/.tox/py36-test
SKIPPED: InterpreterNotFound: python3.6
py36-rapidjson create: /home/mlenders/Repositories/RIOT-OS/riotctrl/.tox/py36-rapidjson
SKIPPED: InterpreterNotFound: python3.6
py37-test inst-nodeps: /home/mlenders/Repositories/RIOT-OS/riotctrl/.tox/.tmp/package/2/riotctrl-0.5.0.zip
py37-test installed: attrs==21.2.0,coverage==5.5,importlib-metadata==4.8.1,iniconfig==1.1.1,packaging==21.0,pexpect==4.8.0,pluggy==1.0.0,psutil==5.8.0,ptyprocess==0.7.0,py==1.10.0,pyparsing==2.4.7,pytest==6.2.5,pytest-cov==2.12.1,riotctrl @ file:///home/mlenders/Repositories/RIOT-OS/riotctrl/.tox/.tmp/package/2/riotctrl-0.5.0.zip,toml==0.10.2,typing-extensions==3.10.0.2,zipp==3.5.0
py37-test run-test-pre: PYTHONHASHSEED='1288904933'
py37-test run-test: commands[0] | pytest -m 'not rapidjson' -s
==================================================================================================================================================== test session starts =====================================================================================================================================================
platform linux -- Python 3.7.11, pytest-6.2.5, py-1.10.0, pluggy-1.0.0 -- /home/mlenders/Repositories/RIOT-OS/riotctrl/.tox/py37-test/bin/python
cachedir: .tox/py37-test/.pytest_cache
rootdir: /home/mlenders/Repositories/RIOT-OS/riotctrl, configfile: setup.cfg, testpaths: riotctrl
plugins: cov-2.12.1
collected 25 items / 1 deselected / 24 selected                                                                                                                                                                                                                                                                              

riotctrl/tests/ctrl_test.py::test_riotctrl_application_dir PASSED
riotctrl/tests/ctrl_test.py::test_riotctrl_curdir PASSED
riotctrl/tests/ctrl_test.py::test_riotctrl_flash PASSED
riotctrl/tests/ctrl_test.py::test_running_echo_application sh -c 'echo $$ > /tmp/tmp29cb9c42; exec ./ctrl.py ./echo.py'
Starting RIOT Ctrl
This example will echo
Hello Test 0
Starting RIOT Ctrl
This example will echo
Hello Test 0
Hello Test 1
Hello Test 1
Hello Test 2
Hello Test 2
Hello Test 3
Hello Test 3
Hello Test 4
Hello Test 4
Hello Test 5
```

Reply:
Ah, thanks! Good to know :)

Reply:
You have to specify the testenv and the complete test path:

```
tox -e test riotctrl/tests/ctrl_test.py::test_riotctrl_application_dir
```